src/header.rs

```diff
@@ -0,0 +1,243 @@
+use error::Mp3Error;
+use tables::*;
+
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
All these enums seem like a perfect opportunity to use the [`strum_macros`](https://crates.io/crates/strum_macros) crate to get proper `Display` impls for "free".

Reply:
See comment above: `strnum_macros` depends on the entire `proc_macro` crate tree. That's quite many dependencies. `Display` impls per se are useful though. But we can tackle that question later on.

Reply:
Fair enough, let's do our best to avoid "heavy" deps then! 👍